mbtiles/scripts/cli.py

```diff
@@ -189,6 +204,12 @@ def resolve_inout(
 @click.option(
     "--progress-bar", "-#", default=False, is_flag=True, help="Display progress bar."
 )
+@click.option(
+    "--cutline",
+    callback=extract_features,
+    default=None,
+    help="A GeoJSON feature collection to be used as a cutline. Only source pixels within the cutline features will be exported.",
```

Comment:
Maybe this should say "Path to a GeoJSON FeatureCollection ..."?

Reply:
Yup sounds better.